packages/terra-status-view/src/terra-dev-site/doc/status-view/StatusView.1.doc.jsx

```diff
@@ -10,7 +8,20 @@ import { name } from '../../../../package.json';
 import StatusViewSrc from '!raw-loader!../../../../src/StatusView';
 
 // Example Files
-import StatusViewTemplate from '../example/StatusViewTemplate';
+import NoPropsExample from '../example/NoPropsExample';
+import NoPropsExampleSrc from '!raw-loader!../example/NoPropsExample';
```

Comment:
These raw-loader paths need updated to no show compiled `lib` code as output. Here is an example:

https://github.com/cerner/terra-core/blob/master/packages/terra-action-footer/src/terra-dev-site/doc/action-footer/Standard.a.doc.jsx#L14

Reply:
Updated: https://github.com/cerner/terra-core/pull/2130/commits/b6cc3d82c58603e2146e1eb06400d6dd8dbc51ca